static/app/api.tsx

```diff
@@ -345,25 +370,33 @@ export class Client {
     /**
      * Called when the request completes with a 2xx status
      */
-    const successHandler = (responseData: any, textStatus: string, xhr: JQueryXHR) => {
+    const successHandler = (
+      resp: ResponseMeta,
+      textStatus: string,
+      responseData: any
```

Comment:
I changed the parameter order here to match with errorHandler

Reply:
Is this a private function?

Reply:
Yeah, it's only defined inside of `request`